src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,156 @@
+package duke;
+
+// import duke packages
+import duke.command.Deadline;
+import duke.command.Event;
+import duke.command.Todo;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.util.Parser;
+import duke.util.Storage;
+import duke.util.Ui;
+
+// import java packages
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * A chatbot that stores tasks given by a user.
+ */
+public class Duke {
+    private Storage storage;
+    private TaskList cmdList;
```

Comment:
`cmdList` might be hard to pronounce and might not be the best choice of name according to the [Code Quality Guideline](https://nus-cs2103-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html). 
Perhaps changing to `commandList` would be clearer?